modules/LBSTransientSolver/IterativeOperations/lbts_settransientsource.cc

```diff
@@ -115,14 +115,17 @@ void lbs::TransientSolver::
           //====================== Apply accross-groupset fission
           if (fission_avail and apply_ags_fission_src)
           {
+            const double spec = options.use_precursors?
```

Comment:
if precursors are UNIMPORTANT in my physical problem (say fast burst), should I, as a user, input data for precursors? No. So use_precursors will be F. So chi will be used. Who gives chi? I do, as a user. However, I should know that I will have to enter prompt values as if they were total (hence, I enter chi_p as chi, nu_p as nu, and so on). IS this how you envision fast burst to be run?

Reply:
@ragusa I've thought about this a bit and there are numerous ways to handle it. I will caveat this by saying that when we use a production matrix (my next PR), this check goes away. Nonetheless, this is still an important discussion when considering inputs. Currently, there are two ways to specify a fast burst where only prompt neutrons are considered.

1. Supply $\chi$ and $\nu$ with the prompt values as you suggested.
2. Specify the problem with delayed neutron precursors and use $\nu^d = 0$ for all groups.
    a. This would lead to some unnecessary computation of delayed neutron terms

The first method makes sense and the second is hacky, so it would be good to make this more robust. I think the best input options would be:

1. See 1. above.
2. Supply *only* $\nu^p$ and $\chi^p$. This will require once again re-configuring the checks done on cross sections.
3. Supply all prompt/delayed data and set `options.use_precursors = false`.

Thoughts?

Reply:
Fine with the issue creation. Please make, if you don't object, the name `spec` here be any of the following:
- `spec_g`
- `spectrum_g`
- `chi_g`

Any of these would suffice.

Reply:
That line goes away in the next PR I'll have that implements the production matrix. I can change it or we can just leave it. Up to you @Naktakala

Reply:
Ok